Api/ClientApp/src/pages/User.jsx

```diff
@@ -35,10 +46,31 @@ export function User() {
     event.preventDefault()
     console.log('Dismiss:')
   }
+
+  function getUser() {
+    fetch(`/api/Users/${displayName}`)
+      .then(response => response.json())
+      .then(apiData => {
+        setUser(apiData)
+      })
+  }
+
+  function getCharacter(characterVariableName) {
+    fetch(`/api/Characters/${characterVariableName}`)
```

Comment:
I feel like this get characters function should be something exported from the characters component, this code shouldn't be duplicated into everywhere that needs to get a character.